frontend/packages/citizen-frontend/src/localization/fi.tsx

```diff
@@ -770,14 +770,15 @@ export default {
           DAYCARE: function FeeInfoTextDaycare() {
             return (
               <P>
-                Kunnallisen varhaiskasvatuksen asiakasmaksut määräytyvät
-                prosenttiosuutena perheen bruttotuloista. Maksut vaihtelevat
-                perheen koon ja tulojen sekä varhaiskasvatusajan mukaan
-                maksuttomasta varhaiskasvatuksesta enintään 288 euron
-                kuukausimaksuun lasta kohden. Perhe toimittaa tuloselvityksen
-                bruttotuloistaan tuloselvityslomakkeella, viimeistään kahden
-                viikon kuluessa siitä, kun lapsi on aloittanut
-                varhaiskasvatuksessa.
+                Kunnallisen varhaiskasvatuksen asiakasmaksut ja palvelusetelin
+                omavastuuosuus määräytyvät prosenttiosuutena perheen
+                bruttotuloista. Maksut vaihtelevat perheen koon ja tulojen sekä
+                varhaiskasvatusajan mukaan maksuttomasta varhaiskasvatuksesta
+                enintään 288 euron kuukausimaksuun lasta kohden.
```

Comment:
Supernit: do we want to mix € and "euro" in the same message?

Reply:
Thanks for feedback @mikkopiu! In my opinion the present one is pretty good.